### PR TITLE
Teambuilder: Set format when importing Pokepaste

### PR DESCRIPTION
**Pull request by Karthik99999**
I thought this would be a cool thing to add to go along with #1882. Code will have to be updated once actual notes gets implemented, but I imagine it'll be a while before those come out.

### CODE REVIEW

File: js/client-teambuilder.js [hunk 1]
```diff
@@ -1289,12 +1289,14 @@
 					success: function (data) {
 						if (/^https?:\/\/pokepast\.es\/.*\/json\s*$/.test(url)) {
 							var teamData = JSON.parse(data);
-							Storage.activeSetList = self.curSetList = Storage.importTeam(teamData.paste);
+							var format = window.BattleFormats && window.BattleFormats[toID(teamData.notes.slice(8))];
```
Comment (Zarel) [inline js/client-teambuilder.js:1292]: It's probably worth checking if the first 8 characters of `teamData.notes` is in fact `"Format: "`.

Also, can we slice it only to the next linebreak? I think both of these are probably useful future-proofing.

File: js/client-teambuilder.js [hunk 2]
```diff
@@ -1289,8 +1289,12 @@
 					success: function (data) {
 						if (/^https?:\/\/pokepast\.es\/.*\/json\s*$/.test(url)) {
 							var teamData = JSON.parse(data);
-							var format = window.BattleFormats && window.BattleFormats[toID(teamData.notes.slice(8))];
-							if (format) self.changeFormat(format.id);
+							var notes = teamData.notes;
+							if (notes.startsWith('Format: ')) {
+								var formatid = toID(notes.slice(8, notes.indexOf('\n')));
```
Comment (Zarel) [inline js/client-teambuilder.js]: I think this breaks if `notes` doesn't contain `\n`.

Comment (Karthik99999) [reply js/client-teambuilder.js]: i fixed that in my latest commit

**Comment by Zarel** [comment]
Thanks!

**Pull request merged**